hypercall/include/c/x64/amd/mv_reg_t.h

```diff
@@ -33,7 +33,25 @@
  */
 enum mv_reg_t
```

Comment:
Dummy should not be there. It was only added to make the compiler happy. Please add all of them in the spec so that we don't have to touch the file again. Also, it should look more like this (although in C and not C++). 
https://github.com/Bareflank/hypervisor/blob/master/syscall/include/x64/amd/bf_reg_t.hpp#L39

The integer type should be uint64_t. C doesn't support defining the underlying type, but we should still be casting the literals.

Reply:
Ok, my next patch has the code for this done already, so when you rebase you can just use that code instead of this.